themes/shezaan/shezaan.theme.bash

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+SCM_THEME_PROMPT_DIRTY=" ${red}✗"
+SCM_THEME_PROMPT_CLEAN=" ${bold_green}✓"
+SCM_THEME_PROMPT_PREFIX=" |"
+SCM_THEME_PROMPT_SUFFIX="${green}|"
+
+GIT_THEME_PROMPT_DIRTY=" ${red}✗"
+GIT_THEME_PROMPT_CLEAN=" ${bold_green}✓"
+GIT_THEME_PROMPT_PREFIX=" ${green}|"
+GIT_THEME_PROMPT_SUFFIX="${green}|"
+
+RVM_THEME_PROMPT_PREFIX="|"
+RVM_THEME_PROMPT_SUFFIX="|"
+
+myip=`$BASH_IT/lib/get_ip.py`
```

Comment:
I would also caution about including backticks-defined variable without wrapping it in the `if` statement, due to the additional latency introduced when executing the script – whether or not the value of the variable is used later.

This seems like a good candidate to have a global enable/disable flag (and off by default), something like `SCM_THEME_SHOW_MY_IP=true|false`